sdk/core/Azure.Core/samples/Events.md

```diff
@@ -0,0 +1,190 @@
+# Azure.Core Event samples
+
+**NOTE:** Samples in this file only apply to packages following the
+[Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html).
+The names of these packages usually start with `Azure`.
+
+Most Azure client libraries offer both synchronous and asynchronous methods for
+calling Azure services.  You can distinguish the asynchronous methods by their
+`Async` suffix.  For example, `BlobClient.Download` and `BlobClient.DownloadAsync`
+make the same underlying REST call and only differ in whether they block.  We
+recommend using our async methods for new applications, but there are perfectly
+valid cases for using sync methods as well.  These dual method invocation
+semantics work great, but require a little extra care when writing event handlers.
+
+The `SyncAsyncEventHandler` is a delegate used by events in Azure client
```

Comment:
For event handler and event args ... link to the source or docs so folks can learn more?

Reply:
There are no docs (yet) and I don't think the source adds any value compared to what's here (i.e., [Handler](https://github.com/Azure/azure-sdk-for-net/blob/f27c837934e9eb68a74aca4e1a5d3fabbe33d40c/sdk/core/Azure.Core/src/SyncAsyncEventHandler.cs) and [Args](https://github.com/Azure/azure-sdk-for-net/blob/f27c837934e9eb68a74aca4e1a5d3fabbe33d40c/sdk/core/Azure.Core/src/SyncAsyncEventArgs.cs)).